example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -378,29 +379,42 @@ describe('The S3 Ingest Granules workflow', () => {
 
   describe('the CopyToGlacier task', () => {
     let lambdaOutput;
+    let copyGlacierBeforeAllFailed = false;
 
     beforeAll(async () => {
-      lambdaOutput = await lambdaStep.getStepOutput(workflowExecutionArn, 'copy_to_glacier');
+      try {
+        lambdaOutput = await lambdaStep.getStepOutput(workflowExecutionArn, 'copy_to_glacier');
+      } catch (error) {
+        copyGlacierBeforeAllFailed = true;
+        throw error;
+      }
+      orcaStepIncludedInWorkflow = lambdaOutput !== null;
```

Comment:
:makes a face: 

This feels really really not great.    Obviously it's a temp workaround.....

Reply:
Which part is not great, specifically? Disabling parts of the test conditionally? I agree but can't think of another quick workaround